app/portainer/components/be-feature-indicator/be-feature-indicator.html

```diff
@@ -1,4 +1,5 @@
 <a class="be-indicator" href="{{ $ctrl.url }}" target="_blank" rel="noopener" ng-if="$ctrl.limitedToBE">
+  <span ng-transclude="children"></span>
```

Comment:
if we have only one, than we can use here just `ng-transclude`, in the component definition change `transclude: object` to `transclude: true` and in the browse button just pass the children without a wrapping element (`inidicator-children`). Sort of like the difference between react's children to a render prop